src/main/java/org/jvnet/hudson/test/TestPluginManager.java

```diff
@@ -113,6 +129,19 @@ private TestPluginManager() throws IOException {
         return names;
     }
     
+    /**
+     * Install a plugin from the resources directory.
+     * @param pluginName The plugin name.
+     * @throws IOException Error copying plugin.
+     */
+    public void installResourcePlugin(String pluginName) throws Exception {
+        URL res = getClass().getClassLoader().getResource("plugins/" + pluginName);
```

Comment:
`getClass().getClassLoader().getResource(…)` is generally an antipattern. What is this `Class`? Might it have been subclassed in a different package? If you are sure that the class loader you want is that which loaded `TestPluginManager`, then say so: `TestPluginManager.class.getClassLoader().getResource(…)`.